### DISCUSSION TITLE
How to resume model training from latest checkpoint in spacy?

### DISCUSSION THREAD

**Discussion by karndeepsingh** [explosion/spaCy#9187] [Help: Coding & Implementations] [answered]
Hi,
I have trained ner and relation model together for 4000 iterations but I want to continue my training for few more iterations from the latest checkpoints that have been stored. Please, help to understand how I can resume my model training from latest checkpoints.

**Comment by polm** [comment, marked as answer]
We actually just had a thread about this last week: #9157. While this is kind of possible, unfortunately it's not really straightforward right now. For how to do it see #8176.

**Reply by karndeepsingh** [reply]
@polm Please also help me to find a command or way to evaluate a trained model with its best weights on test dataset directly??

Thanks

**Reply by polm** [reply]
You can use [`spacy evaluate`](https://spacy.io/api/cli#evaluate).
